addons/4d/doc_classes/TetraMaterial4D.xml

```diff
@@ -19,6 +19,9 @@
 		<member name="albedo_source" type="int" setter="set_albedo_source" getter="get_albedo_source" enum="TetraMaterial4D.TetraColorSource" default="0">
 			The albedo source of the material, an enum specific to tetrahedral meshes. Can be a single color, per-vertex, per-cell, per-cell UVW coordinates, 4D texture, or a combination of single color and the other color sources.
 		</member>
+		<member name="texture" type="Texture3D" setter="set_texture" getter="get_texture">
+			The albedo texture of the material as a Texture3D. Used by the cross-section renderer. Used when albedo_source is [constant TETRA_COLOR_SOURCE_TEXTURE4D_ONLY] or [constant TETRA_COLOR_SOURCE_TEXTURE4D_AND_SINGLE].
```

Comment:
That's actually not what the Texture4D color source is meant to be used for. Texture4D is when you have a 4D texture which directly maps 4D coordinates to a 4D texture without using the cell UVW map. The `TETRA_COLOR_SOURCE_CELL_UVW_ONLY` and `TETRA_COLOR_SOURCE_CELL_UVW_AND_SINGLE` options are for using a cell UVW map to map cells onto a 3D texture.

Reply:
Ah ok, fixed. Is texture 4D a thing? Not sure I've seen that as a supported type anywhere. Might need to be a shader or a sprite sheet type thing with some manual sampling.

Reply:
It's not a thing in Godot yet, but can be done with shaders. See https://www.youtube.com/watch?v=Ir8oft_qAMQ&t=3m35s